frontend/src/components/Collections/components/DeleteCollection/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, H6, Intent } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import { navigate } from "@reach/router";
+import React, { FC, useState } from "react";
+import { ROUTES } from "src/common/constants/routes";
+import { Collection } from "src/common/entities";
+import { useDeleteCollection } from "src/common/queries/collections";
+import Alert from "src/components/Alert";
+
+interface Props {
+  id: Collection["id"];
+}
+
+const DeleteCollection: FC<Props> = ({ id }) => {
+  const [deleteMutation] = useDeleteCollection(id);
+
+  const handleDelete = () => {
+    deleteMutation(id);
```

Comment:
I'm assuming we need to await here until the DELETE call succeeds before we navigate, otherwise we might still see the collection on My Collections page?

e.g., 

```ts
const handleDelete = async () => {
  await deleteMutation(id);
  ...
}
```